.github/workflows/pull_request.yml

```diff
@@ -36,9 +31,9 @@ jobs:
     - name: Check formatting
       run: npm run format:check
       working-directory: apps/super-app/
-    - name: Build
-      run: npm run build
-      working-directory: apps/super-app/
     - name: Test
       run: npm test
       working-directory: apps/super-app/
+    - name: Build
+      run: npm run build
+      working-directory: apps/super-app/
```

Comment:
Why the reordering?

Reply:
I thought it makes more sense to build once testing is finished!